eZ/Publish/SPI/Repository/Tests/Decorator/LanguageServiceDecoratorTest.php

```diff
@@ -108,7 +108,7 @@ public function testLoadLanguageByIdDecorator()
         $serviceMock = $this->createServiceMock();
         $decoratedService = $this->createDecorator($serviceMock);
 
-        $parameters = ['random_value_5ced05ce0e4f86.21411523'];
+        $parameters = [100];
 
         $serviceMock->expects($this->once())->method('loadLanguageById')->with(...$parameters);
```

Comment:
You can probably pass `100` directly here, this code was probably generated in more or less automated way, no one said it should be the same for test :)